src/App.js

```diff
@@ -1,7 +1,70 @@
 import Background from "./assets/Background.svg";
 import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
 
 function App() {
+  const LOGIN_API_URL = "http://localhost:3000/auth/login";
+
+  const CREATE_USER_URL = "http://localhost:3000/users";
+
+  const navigate = useNavigate();
+
+  async function user(mail, pass) {
+    const response = await fetch(CREATE_USER_URL, {
```

Comment:
If you're going to make several API requests from different components, it makes it easier to move the common code out of the component. 
It's a common pattern to have POSTs that send a body (like you're doing twice here), so it helps to have a generic `post` and `get` function, where you can just pass in the path and body, something like:
```
const response = await post(`/users`, {
  email: mail,
  password: pass,
})
```
and the `post` function deals with setting the headers, preparing the body, setting the API base URL, etc, so you don't have to do that every time.

One way to do this is with [custom hooks](https://react.dev/learn/reusing-logic-with-custom-hooks). You could have a `useAPI` hook, for example.